storybook/webpack.config.ts

```diff
@@ -151,10 +151,25 @@ export default ({ config }) => {
   // Required for the storysource storybook addon
   config.module.rules.push(storybookSource)
 
+  config.module.rules.push({
+    test: /^((?!\.stories).)*\.tsx?$/, // match all .tsx files other than .stories.tsx files
```

Comment:
This was because I was getting a lot of typescript errors on the *.stories.tsx files. We don't need documentation on the stories themselves (that would be very meta), so I exclude them.